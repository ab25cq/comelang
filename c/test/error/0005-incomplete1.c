/*
PATTERN: incomplete
PATTERN: c:10:
*/

struct X;

int main()
{
	struct X x;
	return 0;
}

struct X {int v;};
