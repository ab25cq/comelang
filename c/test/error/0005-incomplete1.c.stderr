cannot use incomplete type in this context at test/error/0005-incomplete1.c:10:5
    struct X x;
    ^
